src/api/admin.ts

```diff
@@ -13,20 +13,31 @@ export interface EventInput {
   price: number;
 }
 
-export interface VenueInput {
+interface Question {
```

Comment:
Can we use the existing question type?

Reply:
Actually name is purely internal, don't think it should be part of the form (can slugify the text again)